### DISCUSSION TITLE
2024/04/05/caddy-static-website/

### DISCUSSION THREAD

**Discussion by giscus** [hmsjy2017/blog-comment#8] [Announcements]
# 2024/04/05/caddy-static-website/

前言与 Nginx 相比，语法简洁和自动申请 TLS 证书是 Caddy 的两大优势。使用 Caddy 建站对入门者来说更为容易。 本文简述使用 Caddy 搭建静态网站的步骤。 安装对于 Ubuntu 和 Debian 系统，只需几条命令即可安装。 1234sudo apt install -y debian-keyring debian-archive-keyring apt-transpor

https://blog.iamsjy.com/2024/04/05/caddy-static-website/

**Comment by drunken-beholder** [comment]
施斩太有实力了
